nf_core/__main__.py

```diff
@@ -1056,6 +1064,71 @@ def local(ctx, keywords, json, dir):  # pylint: disable=redefined-builtin
         sys.exit(1)
 
 
+# nf-core subworkflows update
+@subworkflows.command()
+@click.pass_context
+@click.argument("subworkflow", type=str, required=False, metavar="subworkflow name")
+@click.option(
+    "-d",
+    "--dir",
+    type=click.Path(exists=True),
+    default=".",
+    help=r"Pipeline directory. [dim]\[default: current working directory][/]",
+)
+@click.option("-f", "--force", is_flag=True, default=False, help="Force update of subworkflow")
+@click.option("-p", "--prompt", is_flag=True, default=False, help="Prompt for the version of the subworkflow")
+@click.option("-s", "--sha", type=str, metavar="<commit sha>", help="Install subworkflow at commit SHA")
+@click.option("-a", "--all", is_flag=True, default=False, help="Update all subworkflow installed in pipeline")
+@click.option(
+    "-x/-y",
+    "--preview/--no-preview",
+    is_flag=True,
+    default=None,
+    help="Preview / no preview of changes before applying",
+)
+@click.option(
+    "-D",
+    "--save-diff",
+    type=str,
+    metavar="<filename>",
+    default=None,
+    help="Save diffs to a file instead of updating in place",
+)
+@click.option(
+    "-r",
+    "--recursive",
```

Comment:
I would prefer to use `--no-prompt`. I associate `--force` to the `install` command if the module is already installed, it sounds more like "force something that can't be done". But I also found `--no-prompt` super confusing with the current `--prompt` which is prompting for the git sha. 
Also, using `--no-prompt` should eliminate all prompts, and this command has lots, which are not always easy to avoid. For example, when the module is not found in the branch; when the option to save differences to a filename is set to True, but the filename is not provided...

Reply:
how about `--update-deps`?

Reply:
I like this name more! Thanks :D